all/pom.xml

```diff
@@ -113,4 +113,34 @@
             <scope>provided</scope>
         </dependency>
     </dependencies>
+
+    <profiles>
+        <profile>
+            <id>autoInstallAll</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.day.jcr.vault</groupId>
+                        <artifactId>content-package-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>install-package</id>
+                                <goals>
+                                    <goal>install</goal>
+                                </goals>
+                                <configuration>
+                                    <targetURL>http://${aem.host}:${aem.port}${aem.contextPath}/crx/packmgr/service.jsp</targetURL>
```

Comment:
I'd change this to `${quickstart.url}/crx/packmgr/service.jsp`, the parent project in this repo defines this extra parameter at https://github.com/adobe/aem-core-cif-components/blob/master/parent/pom.xml#L95